portfolio/src/main/webapp/script.js

```diff
@@ -14,9 +14,31 @@
 
 
 
-// Adds "Hello Andrew" to page 
+// Adds "Hello Andrew" 
 async function getGreeting() {
   const response = await fetch('/data');
   const greeting = await response.text();
-  document.getElementById('greeting-container').innerText = greeting;
+  document.getElementById('gen-container').innerText = greeting;
+}
+
+
+// JSON feature
+function getJSON() {
```

Comment:
The camelCase naming convention is to write this as getJson, even if JSON is usually all capitalized